plugins/change-tracker/README.md

```diff
@@ -0,0 +1,22 @@
+# Strapi plugin change-tracker
+
+A custom plugin for tracking which content changes are published and which aren't.
+This plugin works by writing all content changes to db, and then it compares their creation times to the last deployment's time.
+
+Changes ovr a month old are deleted automatically.
```

Comment:
Does this mean that plugin deletes unpublished changes after month? If so, I would leave the deleting of content solely to authors instead of forcing it via code.

Reply:
Oh bad wording by on my part. Meant to say that the change-data, not the content itself, is deleted after one month. But I could add logic that only deployed changes older than one month are deleted. 

TL;DR; This doesn't affect changed content in anyway, only the data about the change (that we ourself gather).